tool/plugin/lib/lint.dart

```diff
@@ -80,8 +80,6 @@ class LintCommand extends Command {
       'com.android.annotations.NonNull',
       'io.netty.',
       'javax.annotation.Nullable',
-      'org.apache.commons.lang3.StringUtils',
```

Comment:
Let's keep this first exclusion - we're already using `org.apache.commons.lang.StringUtils` in the codebase.

Reply:
done